alacritty/build.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Joe Wilm, The Alacritty Project Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+fn main() {
+    println!("cargo:rustc-env=GIT_HASH={}",
+             rustc_tools_util::get_commit_hash().unwrap_or_default())
```

Comment:
I'm not sure if we want to `unwrap_or_default` this. I think an `expect("getting commit hash")` might be better because it would be easier to catch errors?

I'm slightly on the edge, but I think it's better than printing no commit hash and not noticing.